codebase/superdarn/src.bin/tk/tool/make_raw.1.8/make_raw.c

```diff
@@ -214,11 +219,27 @@ int main (int argc,char *argv[]) {
     exit(-1);
   }
 
+  command[0]=0;
+  n=0;
```

Comment:
Is n defined somewhere in this file? Noticed below you did `int c,n;' but now here?

Reply:
It's defined on line 114. I think it's ok to leave this code as-is
https://github.com/SuperDARN/rst/blob/967553525ea92ab2f64058ed519c5ddfb52f5ec5/codebase/superdarn/src.bin/tk/tool/make_raw.1.8/make_raw.c#L114